src/main/java/com/gregtechceu/gtceu/api/item/OpticalPipeBlockItem.java

```diff
@@ -26,18 +20,6 @@ public OpticalPipeBlockItem(OpticalPipeBlock block, Properties properties) {
         super(block, properties);
     }
 
-    @Override
-    public void appendHoverText(ItemStack stack, @Nullable Level level, List<Component> tooltip,
-                                TooltipFlag isAdvanced) {
-        tooltip.add(Component.translatable("block.gtceu.normal_optical_pipe.tooltip"));
```

Comment:
why remove the tooltip?

Reply:
Because this will result in duplicate tooltips, tooltips in the format of descriptionId.tooltip will be automatically attached.